### DISCUSSION TITLE
Constant name to be uppercase

### DISCUSSION THREAD

**Discussion by MarcTCruz** [vlang/v#11772] [Ideas]
For easier distinguishing from variables, constant names should be uppercase allowing underlines.

It makes the code a lot easier to read and is a basic convention among most languages and good practices.

**Comment by danieldaeschle** [comment]
There is also this: https://twitter.com/v_language/status/1245756980771336192

Didn't understand why it was changed.

**Comment by ntrel** [comment]
https://github.com/vlang/v/discussions/7610#discussioncomment-269080

**Comment by kitech** [comment]
https://github.com/vlang/v/issues/8005
